src/containers/Masthead/components/Navigation.tsx

```diff
@@ -142,6 +146,14 @@ const Navigation = () => {
                     </StyledLogoDiv>
                     <SavedSearchDropdown onClose={closeMenu} />
                   </LeftContent>
+                  <LinkWrapper>
+                    <SafeLink target="_blank" to="https://edndla.zendesk.com/hc/no">
+                      Zendesk
+                    </SafeLink>
+                    <SafeLink target="_blank" to="https://kvalitet.ndla.no/">
+                      Kvalitaisen
```

Comment:
Seriøst?

Reply:
Ja 😅

Reply:
Og det er ikkje behov for å ha dette i phrases?

Reply:
Ser ikke helt hvordan vi skal oversette det til engelsk, men kan dobbeltsjekke hva de ønsker der 👍